Tests/Spec/server_scenario_08_upgrade_spec.rb

```diff
@@ -15,9 +15,11 @@
   it { should have_property_value('InstallLocation', :type_string, "C:\\Program Files\\Octopus Deploy\\Octopus\\") }
 end
 
-describe windows_registry_key('HKEY_LOCAL_MACHINE\Software\Octopus\OctopusServer\OctopusServer') do
+describe file('C:/ProgramData/Octopus/OctopusServer/Instances/OctopusServer.config') do
```

Comment:
Feels like we might need a check here to ensure we have deleted the registry entry?
Then again, is this the right spot for that test 🤔.

Reply:
The registry will be left in place for migrated data, as a rollback option

Reply:
Ahh, cool. Worth a comment on each of the tests to make that explicit?